pyproject.toml

```diff
@@ -25,8 +25,8 @@ select = ["E", "W", "F", "C", "N", "R", "D", "H"]
 # Ignore W503, E501 because using black creates errors with this
 # Ignore D107 Missing docstring in __init__
 ignore = ["W503", "E501", "D107"]
-# D100, D101, D102, D103: Ignore missing docstrings in tests
-per-file-ignores = ["tests/*:D100,D101,D102,D103,D104,D205,D212,D415,DCO010", "src/chrony.py:DCO060"]
+# Ignore some rules that conflict with the test docstring format and the data classes in chrony.py
+per-file-ignores = ["tests/*:D205,D212,D415,DCO010,DCO020,DCO030", "src/chrony.py:DCO060"]
```

Comment:
Can you add the DCO060 ignore directly in the `src/chrony.py` with an explanatory comment?

Reply:
Okay, I've moved this rule to `src/chrony.py`. Unfortunately, it can't be disabled on the exact line as mentioned in the docstring, so I have to disable it for the entire file, thanks!